src/app.py

```
@@ -0,0 +1,7 @@
+if __name__ == '__main__':
+  # 공백 2개인데 뭐라고 하는지 궁금하네.
+  print('Hello World')
+
+  a = 10
+  b = a*2
+  d = a + d
```

Ruff (0.8.2)

Undefined name `d`

(F821)